assets/scss/modules/_modal-dialog.scss

```diff
@@ -0,0 +1,141 @@
+[Using the dialog role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_dialog_role),
+[Making an accessible dialog box - NCZOnline](https://www.nczonline.net/blog/2013/02/12/making-an-accessible-dialog-box/),
+[HTML5 Accessibility Chops: hidden and aria-hidden](https://www.paciellogroup.com/blog/2012/05/html5-accessibility-chops-hidden-and-aria-hidden/),
+
+
+Markup:
+<div class="modal-dialog">
+    <div class="modal-dialog__inner">
+       <div id="content-1" class="modal-dialog__content" data-ga-open-event="::" tabindex="1" role="dialog" aria-labelledby="heading-id" aria-describedby="message-id">
+         <div id="message-id">
+           <h2 id="heading-id">content 1 heading</h2>
+           <p>&lt;!-- content here --></p>
+           <button role="button" class="button button" data-journey-click="::"
+             data-modaldialog-action="close" tabindex="1">Close</button>
+            <button role="button" class="button button--secondary" data-journey-click="::" data-modaldialog-action="open" data-modaldialog-target="content-2" tabindex="1">Open other dialog</button>
+       </div>
+       </div>
+       <div id="content-2" class="modal-dialog__content" data-ga-open-event="::" tabindex="1" role="dialog" aria-labelledby="heading-id-2" aria-describedby="message-id-2" hidden>
+         <div id="message-id-2">
+           <h2 id="heading-id-2">content 2 heading</h2>
+           <p>&lt;!-- content here --></p>
+           <a href="#" data-modaldialog-action="close" tabindex="1" data-journey-click="::">Close</a>
+         </div>
+       </div>
+   </div>
+ </div>
+
+
+Style guide: Modal Dialog
+*/
+$mask-opacity: rgba(89, 89, 89, .7); //#595959;
+$modal-dialog-background: rgba(255, 255, 255, 1);
+$modal-dialog-border: em(5) solid rgba(0, 0, 0, 1);
+$modal-dialog-box-shadow: 0 em(2) 0 #6f777b;
+$modal-dialog-yellow: $attorney-yellow; //#ffbf47;
+// set-square-clip($for-ie, $px-value)
+//
+// Create square clip rule with rect style dimensions.
+//
+// Rule compatible in IE8+, MSEdge, Safari 9.0.3, Google Chrome 48.0.2564.1, Firefox 44.0.2,
+//
+// $for-ie   - boolean: generate IE compatible rule with no commas (default 'false').
+// $px-value - integer: size of square clip dimension.
+//
+// Styleguide: sass.mixins.set-square-clip
+@mixin set-square-clip($for-ie: false, $px-value: 0) {
+  @if $for-ie == true {
+    clip: rect(em($px-value) em($px-value) em($px-value) em($px-value));
+  } @else {
+    clip: rect(em($px-value), em($px-value), em($px-value), em($px-value));
+  }
+}
+body {
+  &.scroll-off {
+    position: fixed;
+    width: 100%;
+  }
+}
+.modal-dialog {
+  &.modal-dialog--hidden {
+    @include set-square-clip(true, 1);
+    @include set-square-clip(false, 1);
+    border: 0;
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+    z-index: 0;
```

Comment:
IE stuff should be in the IE stylesheet really or use the IE mixin, I'm not sure whats preferred here